services/ui-src/src/components/cards/EntityCard.tsx

```diff
@@ -0,0 +1,126 @@
+// components
+import { Card } from "components";
+import { Box, Button, Heading, Image, Text } from "@chakra-ui/react";
+// utils
+import { AnyObject } from "types";
+// assets
+import { svgFilters } from "styles/theme";
+import deleteIcon from "assets/icons/icon_cancel_x_circle.png";
+import editIcon from "assets/icons/icon_edit.png";
+import unfinishedIcon from "assets/icons/icon_error_circle.png";
+
+export const EntityCard = ({ entity, ...props }: Props) => {
+  // data to fill in card
+  const data = {
+    category: entity.accessMeasure_generalCategory[0].value,
+    standardDescription: entity.accessMeasure_standardDescription,
+    standardType:
+      entity.accessMeasure_standardType[0].value !== "Other, specify"
+        ? entity.accessMeasure_standardType[0].value
+        : entity["accessMeasure_standardType-otherText"],
+  };
```

Comment:
help me out: will this work on other entity types? Or is it locked to accessMeasure?

Reply:
It's locked to access measure for right now. We'll have to make a custom one for each of the ModalDrawer entity types.